python/mxnet/symbol.py

```diff
@@ -990,12 +1075,30 @@ def simple_bind(self, ctx,
                     type_dict=None,
                     group2ctx=None,
                     **kwargs):
-        """Bind current symbol to get an executor, allocate all the ndarrays needed.
+        """Binds current symbol to get an executor, allocate all the arguments needed.
         Allows specifying data types.
 
-        This function will ask user to pass in an `NDArray` of position
-        they like to bind to, and it will automatically allocate the ndarray
-        for arguments and auxiliary states that user did not specify explicitly.
+        This function simplifies the binding procedure. You need to specify only input data shapes.
+        The function allocates the arguments and auxiliary states that you did not specify
```

Comment:
Suggest rewording as:
"Before binding the executor, the function allocates arguments and auxiliary states that were not explicitly specified."